aquamarine-vm/src/aquamarine_stepper_vm.rs

```diff
@@ -170,13 +163,13 @@ fn make_faas_config(
     }
 }
 
-fn make_raw_outcome(mut result: Vec<IValue>) -> Result<RawStepperOutcome> {
+fn make_outcome(mut result: Vec<IValue>) -> Result<StepperOutcome> {
```

Comment:
I think this function should be moved to `stepper-interface` crate in `aquamarine` repo